keras_cv/metrics/f_scores.py

```diff
@@ -0,0 +1,356 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+import tensorflow.keras as keras
+
+
+class FBetaScore(keras.metrics.Metric):
+    r"""Computes F-Beta score.
+
+    It is the weighted harmonic mean of precision
+    and recall. Output range is `[0, 1]`. Works for
+    both multi-class and multi-label classification.
```

Comment:
@markub3327 cc. @LukeWood 
Is this class compatible with object detection and segmentation task as well?

Reply:
These would probably need a separate implementation I'd think